### ISSUE TITLE
Switchable views between downloads and browser

### ISSUE THREAD

**Issue by CarterSnich** [TachibanaGeneralLaboratories/download-navi#198]
Allow user to check the downloads while browsing. Also, add history to browser.

**Comment by KrasnayaPloshchad** [comment]
I saw [ADM](https://play.google.com/store/apps/details?id=com.dv.adm) have this feature

**Comment by CarterSnich** [comment]
> I saw [ADM](https://play.google.com/store/apps/details?id=com.dv.adm) is already implemented this feature.

Yeah. I actually use ADM before. But then, I encounter some issues and switched to open-source alternatives.

**Comment by KrasnayaPloshchad** [comment]
I also think it’s possible to browse website to get files and manage download tasks simultaneously.
